internal/http/services/owncloud/ocdav/copy.go

```diff
@@ -688,6 +683,38 @@ func (s *svc) prepareCopy(ctx context.Context, w http.ResponseWriter, r *http.Re
 			errors.HandleWebdavError(log, w, b, err) // 412, see https://tools.ietf.org/html/rfc4918#section-9.8.5
 			return nil
 		}
+
+		if utils.IsSpaceRoot(dstStatRes.GetInfo()) {
+			log.Error().Msg("overwriting is not allowed")
+			w.WriteHeader(http.StatusBadRequest)
+			return nil
+		}
+
+		// we must not allow to override mountpoints - so we check if we have write access to the parent. If not this is a mountpoint
```

Comment:
hm, what if we are in a readonly space where we were granted write permissions on a subfolder?

Reply:
the comment is wrong - I only stat it, so view permissions is enough.